modules/git/git.go

```diff
@@ -21,7 +21,7 @@ func GetCurrentBranchName(t testing.TestingT) string {
 // GetCurrentBranchNameE retrieves the current branch name or
 // empty string in case of detached state.
 func GetCurrentBranchNameE(t testing.TestingT) (string, error) {
-	cmd := exec.Command("git", "rev-parse", "--abbrev-ref", "HEAD")
+	cmd := exec.Command("git", "branch", "--show-current")
```

Comment:
This option was introduced in git 2.22, which unfortunately is newer than the default that is installable with Ubuntu 18.04. So I don't think we should switch to this command completely.

Instead, we should do a fallback approach. Either:

- Check the version of git installed and use this if newer than 2.22, or fallback to the other one
- Run `git branch --show-current` and if it fails, fall back to `git rev-parse --abbrev-ref HEAD`.

Reply:
Ah, good idea! Will change. Thanks for the review.